invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/landing_page.less

```diff
@@ -164,6 +164,7 @@
      .content{
          padding-top: 0px !important;
      }
+     -K|--keep-services)
```

Comment:
Oups?

Reply:
Wow, nice catch, thanks! My brain is already on holidays mode ... :sweat_smile: